templates/README.md

```diff
@@ -0,0 +1,33 @@
+# {{library}}
+
+Code: [![Version](https://badge.fury.io/rb/{{library}}.png)](http://badge.fury.io/rb/{{library}})
```

Comment:
From what I could determine by briefly checking, RubyGems names map to GitHub project names

Reply:
Mostly by convention, but yeah.